### PR TITLE
Install HTSeq to make ngi_visualizations work

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -98,6 +98,14 @@ RUN echo 'source("https://bioconductor.org/biocLite.R")' > /opt/packages.r && \
...
+# Install HTSeq
+RUN curl -fsSL https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz -o /opt/HTSeq-0.7.2.tar.gz && \
+    tar xvzf /opt/HTSeq-0.7.2.tar.gz -C /opt/ && \
+    cd /opt/HTSeq-0.7.2  && \
+    python setup.py build  && \
+    python setup.py install  && \
+    rm /opt/HTSeq-0.7.2.tar.gz
+
 # Install NGI Visualizations
 ENV pip install --upgrade git+https://github.com/NationalGenomicsInfrastructure/ngi_visualizations.git
```
Comment (ewels) [inline Dockerfile]: I just noticed that this is broken - should be `RUN` instead of `ENV`.

File: Dockerfile [hunk 2]
```diff
@@ -98,6 +98,14 @@ RUN echo 'source("https://bioconductor.org/biocLite.R")' > /opt/packages.r && \
     Rscript /opt/packages.r && \
     mkdir /usr/local/lib/R/site-library
 
+# Install HTSeq
+RUN curl -fsSL https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz -o /opt/HTSeq-0.7.2.tar.gz && \
```
Comment (ewels) [inline Dockerfile]: This is great, but if HTSeq is on PyPI then why not just install directly with `pip`? Just had a quick go locally and this seems to work:

```bash
RUN pip install numpy && pip install htseq
```

**Review by ewels — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ewels) [reply Dockerfile]: Also it probably shouldn't have a space at the start of the line.. :)

File: Dockerfile (hunk 2 above)
Comment (chuan-wang) [reply Dockerfile]: Thanks! I googled htseq and didn't find such a nice method for installation.

File: Dockerfile (hunk 1 above)
Comment (chuan-wang) [reply Dockerfile]: Also fixed.

File: Dockerfile (hunk 2 above)
Comment (ewels) [reply Dockerfile]: No, their docs are not very clear. But the download address is PyPI (Python Package Index) and any package there should be installable with `pip`. It _should_ automatically install `numpy` as a dependency, but for whatever reason when I tested it the `htseq` installation instead just complained that `numpy` wasn't installed. The sequential installation seems to do the job though..

**Review by ewels — APPROVED** [review]

**Pull request merged**